tests/Issues/AutoImport/Fixture/sub_namespace_dockblock_imported.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
it seems you cut running test in the middle, that cause `.php` file committed, remove this `.php` file

```
rm -rf tests/Issues/AutoImport/Fixture/sub_namespace_dockblock_imported.php
```